shell/initialize/client.js

```diff
@@ -327,70 +226,6 @@ async function render(to, from, next) {
       return next();
     }
 
-    let instances;
-
-    // Call fetch hooks on components matched by the route.
-    await Promise.all(Components.map((Component, i) => {
```

Comment:
Ultimately this could be removed because we'd exit early from this before doing any mutations. I'll explain each section below.